iina/uk.lproj/KeyBinding.strings

```diff
@@ -54,7 +54,7 @@
 "iina.toggle-flip" = "Перемкнути перевертання";
 "iina.toggle-mirror" = "Перемкнути віддзеркалення";
 "iina.toggle-music-mode" = "Перемкнути режим музики";
-"iina.toggle-pip" = "Перемкнути режим «Кадр у кадрі»";
+"iina.toggle-pip" = "Перемкнути режим «Картинка в картинці»";
```

Comment:
Wrong. The red one is used in official Ukrainian localisation of macOS apps (eg., Safari web browser).
<img width="292" alt="Screenshot 2019-05-06 16 38 04" src="https://user-images.githubusercontent.com/36358174/57229211-2eeb1980-701e-11e9-894d-5814d9005f71.png">